src/components/JSONOutput.js

```diff
@@ -8,7 +8,9 @@ const useStyles = makeStyles((theme) => ({
     color: theme.palette.text.primary,
     background: theme.palette.grey[400],
     height: '100%',
-    fontFamily: 'Consolas'
+    fontFamily: 'Consolas',
+    gutterButtom: true,
```

Comment:
Is this a typo?  Should it be `gutterBottom`?

Reply:
I guess a follow up question would be, if there was a typo, do we need that property at all?